docs/customize/configure.md

```diff
@@ -0,0 +1,621 @@
+# Configuration Variables
+
+This document provides a comprehensive list and explanation of the configuration variables available in the `invenio-app-ils` backend. These variables can be set in the configuration file(s) to customize the behavior of your Invenio ILS application.
+
+## REST Configurations
+
+### `ILS_RECORDS_REST_ENDPOINTS`
+
+REST endpoints for various record types.
+
+```python
+ILS_RECORDS_REST_ENDPOINTS = {
+    'books': {
+        'pid_type': 'bookid',
+        'path': '/books',
+        'default_media_type': 'application/json',
+    },
+    'members': {
+        'pid_type': 'memberid',
+        'path': '/members',
+        'default_media_type': 'application/json',
+    },
+}
+```
+
+### `ILS_MAILS_TEMPLATES`
+
+Paths to email templates used for various notifications.
+
+```python
+ILS_MAILS_TEMPLATES = {
+    'loan_due': 'invenio_app_ils/loans/email/loan_due.html',
+    'loan_overdue': 'invenio_app_ils/loans/email/loan_overdue.html',
+}
+```
+
+## Circulation Configuration
+
+### `CIRCULATION_LOAN_TRANSITIONS`
+
+A dictionary defining the possible transitions for loans.
+
+```python
+CIRCULATION_LOAN_TRANSITIONS = {
```

Comment:
this I would describe a bit more as a use case: for example if you need to add a loan in transit state or skip a step in the loan circulation, then you need to modify this variable. Also a bit advanced config.